pandas/core/algorithms.py

```diff
@@ -1041,6 +1041,9 @@ def duplicated(
     -------
     duplicated : ndarray[bool]
     """
+    if hasattr(values, "dtype") and isinstance(values.dtype, BaseMaskedDtype):
```

Comment:
This assumes, that we don't want to add duplicated to the ea interface

Reply:
Might be good to add an issue to discuss this. Kinda unfortunate we need this exception for `BasedMaskedDtype`

Reply:
Yeah not happy about this either, will open a follow up issue to discuss